HW04/siarhei_pratasevich/lib/mentor.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Mentor
```

Comment:
You have a class Mentor and Student, they have common fields, you can add inheritance from a common class to your PR)
Up to you

Reply:
Thanks!) Methods notifications and mark_as_read! for class Student dont expected arguments, but this methods for class Mentor have 1 argument(object student) because mentor may has more of one student. I think its different methods or not?

Reply:
> You have a class Mentor and Student, they have common fields, you can add inheritance from a common class to your PR) Up to you

Thanks!) I added class Participants and added inheritance from this class

Reply:
why this question? You can leave the implementation of these methods where they were, I just suggested adding the simplest inheritance
But it's not necessary, I accept your job.